components/eamxx/mkdocs.yml

```diff
@@ -4,10 +4,9 @@ nav:
   - 'Home': 'index.md'
   - 'User Guide':
     - 'Overview': 'user/index.md'
-    - 'Installation': 'common/installation.md'
```

Comment:
Installation makes no sense for user guide, which is centered on CIME-based usage. The standalone documentation is more for developers.